styles.css

```diff
@@ -437,3 +423,145 @@ form {
   color: #42526e;
   margin-bottom: 300px;
 }
+
+.desktop-menu {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #172b4d;
+}
+
+.desktop-menu-list {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: row;
+  gap: 30px;
+}
+
+@media only screen and (min-width: 768px) {
+  #welcome {
```

Comment:
- [ ] Great design @baezluise, you have done awesome progress but there are a few things in the design that need to be fixed, see the below comments

FIgma design

<img width="440" alt="2" src="https://user-images.githubusercontent.com/77843050/230195373-30ca1552-8cdc-44d4-b4de-74d833a5bc9a.png">
 
- Comparing this design to yours, I think you need to fix some points like the space between the title and the arrow icon, the arrow icon itself, and the text below, please refer to the image to update yours below.

YOUR DESIGN

<img width="556" alt="1" src="https://user-images.githubusercontent.com/77843050/230197850-980028b6-3f5a-4309-9601-c801c395b99b.png">


- The second one is this here below, see how the space between the button and the footer is so small, consider checking the design one more to update that part, please

<img width="1031" alt="4" src="https://user-images.githubusercontent.com/77843050/230198411-4d86aeff-6930-4ddd-bef9-773a353152b3.png">

The figma

<img width="674" alt="3" src="https://user-images.githubusercontent.com/77843050/230198898-feaafe21-ea81-4965-a633-5e375f12d7c5.png">

Reply:
- [ ] These required changes have not been implemented. Kindly attend to it

> * [ ]  Great design @baezluise, you have done awesome progress but there are a few things in the design that need to be fixed, see the below comments
> 
> FIgma design
> 
> <img alt="2" width="440" src="https://user-images.githubusercontent.com/77843050/230195373-30ca1552-8cdc-44d4-b4de-74d833a5bc9a.png">
> 
> * Comparing this design to yours, I think you need to fix some points like the space between the title and the arrow icon, the arrow icon itself, and the text below, please refer to the image to update yours below.
> 
> YOUR DESIGN
> 
> <img alt="1" width="556" src="https://user-images.githubusercontent.com/77843050/230197850-980028b6-3f5a-4309-9601-c801c395b99b.png">
> 
> * The second one is this here below, see how the space between the button and the footer is so small, consider checking the design one more to update that part, please
> 
> <img alt="4" width="1031" src="https://user-images.githubusercontent.com/77843050/230198411-4d86aeff-6930-4ddd-bef9-773a353152b3.png">
> 
> The figma
> 
> <img alt="3" width="674" src="https://user-images.githubusercontent.com/77843050/230198898-feaafe21-ea81-4965-a633-5e375f12d7c5.png">